apps/base-docs/docs/building-with-base/guides/account-abstraction-with-particle-network.md

```diff
@@ -0,0 +1,576 @@
+
+
+## Prerequisites
+
+### Wallet funds
+
+This guide requires you to have ETH on Base Sepolia, which will be used to showcase the execution of a gasless burn transaction.
+
+- To fund your wallet with ETH on Base Sepolia, visit one of the faucets listed on the [Base Faucets](https://docs.base.org/tools/network-faucets) page.
+
+### Familiarity with modern, frontend web development
+
+In this example, you'll be building a React-based application using [`create-react-app`](https://create-react-app.dev). It's recommended that you have some level of familiarity with the basics of working with React.
+
+
+---
+
+
+## Understanding Particle Network
+
+### Wallet-as-a-Service
+
+Particle Network provides a large suite of SDKs centered around the reduction of account-based friction.
+
+In this case, "account-based friction" refers to barriers-to-entry that some Web3 users may face as they onboard into an application and begin managing a wallet.
+
+This friction, in the context of this guide, can be placed within two distinct categories:
+
+1. The login process. Often, decentralized applications that tend to be more consumer-facing prefer login flows that aren't dependent upon a user downloading and managing a traditional wallet, as this can be a pain point for some.
+2. The rigidity of standard accounts. Externally Owned Accounts, or EOAs, are often quite rigid in how they operate. They're secured by one key and limited to a strict range of functions, thus developers (and therefore users) are confined to relatively low-level interaction with applications.
+
+Wallet-as-a-Service (WaaS) aims to solve the first of these two points, the login process. WaaS solutions provide an alternative to standard wallets, typically allowing users to use applications through accounts generated by social logins (such as Google, email, or phone). The interfaces for interacting with these accounts are also often embedded within the applications, resulting in a consistent, application-specific experience.
+
+WaaS providers other than Particle Network include [Web3Auth](https://web3auth.io), [Privy](https://docs.base.org/building-with-base/guides/account-abstraction/intro-to-privy), and [Magic](https://magic.link/), among others.
+
+### Account Abstraction
+
+Particle Network also aims to tackle the second friction point described above: account flexibility.
+
+Account Abstraction refers to a transition away from standard account structures, EOAs, to smart accounts. Smart accounts are contracts that act as a wallet, providing users with an account that feels equivalent to an EOA but is intrinsically programmable (due to it being a smart contract) and thus more flexible.
+
+The most popular modern implementation of Account Abstraction is [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337), which enables Account Abstraction without any consensus-layer protocol changes. It does this through numerous components of supporting infrastructure, including a Bundler and Paymaster.
```

Comment:
Another nit, but do you mind changing to reference-style links?

Reply:
Sure, no problem! Should be good to go now, let me know if the new version works.